client/src/components/game/lab5/pages/PageLayoutActivity/PageLayoutRepair.js

```diff
@@ -0,0 +1,379 @@
+import React, {Component} from 'react';
+import RepairService from '../../../../../services/lab5/RepairService';
+import PageServiceTimer from "../../components/PageServiceTimer";
+import Popup from "../../components/Popup";
+import {navigate} from "@reach/router";
+import {minFont, maxFont} from '../../../../../constants/lab5'
+
+class PageLayoutRepair extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			h1value: null,
+			ulvalue: null,
+			classvalue: null,
+			fontvalue: null,
+			fontfamilyvalue: null,
+		};
+	}
+
+	componentWillMount() {
+		const { data } = this.props;
+		this.setState({
+			h1value: data.h1value,
+			ulvalue: data.ulvalue,
+			classvalue: data.classvalue,
+			fontvalue: data.fontvalue,
+			fontfamilyvalue: data.fontfamilyvalue,
+		});
+	}
+
+	handleSubmit(event) {
+		const { handlers } = this.props;
+		const {
+			h1value,
+			ulvalue,
+			classvalue,
+			fontvalue,
+			fontfamilyvalue
+		} = this.state;
+
+		event.preventDefault();
+		const repair = JSON.stringify({h1value,
+			ulvalue,
+			classvalue,
+			fontvalue,
+			fontfamilyvalue});
+		// Submit a repair entry in the database.
+		RepairService.submitRepair(
+			this.constructor.name, repair
+		);
+
+		// Update the state and close the repair.
+		handlers.updateRepairPageLayout(
+			h1value,
+			ulvalue,
+			classvalue,
+			fontvalue,
+			fontfamilyvalue
+		);
+		handlers.closeRepair();
+		handlers.updatePopup('The repairs have been made.');
+
+		setTimeout(() => {
+			handlers.updatePopup('');
+		}, 5000);
+	}
+
+	changeHandler(event) {
+		const name = event.target.name;
+		const value = event.target.value;
+
+		this.setState({
+			[name]: value
+		});
+	}
+
+	handleNav() {
+		navigate("/Lab5/Game/DementiaAccessible");
+	}
+
+
+	render() {
+		const { visible, handlers, state, data, actions } = this.props;
+		return (
+			<div>
+				<div className= "cognitive_instructions margin-bottom-2">
+					Let's make the changes to make the code accessible. Click 'Repair' to make the appropriate changes.
```

Comment:
I am not sure if this is something we check for in the other labs, but I was able to click the next button without making any repairs. I haven't tried doing this with the other labs, but I just noticed there wasn't a check for that

Reply:
Yup, this is fixed